src/Microsoft.ML.FastTree/Representation/TreeRegressorCollection.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Microsoft.ML.Trainers.FastTree.Internal;
+
+namespace Microsoft.ML.FastTree.Representation
+{
+    /// <summary>
+    /// A list of <see cref="TreeRegressor"/>. To compute the output value of a <see cref="TreeRegressorCollection"/>, we need to compute
+    /// the output values of all trees in <see cref="Trees"/>, scale those values via <see cref="TreeWeights"/>, and finally sum the scaled
+    /// values and <see cref="Bias"/> up.
+    /// </summary>
+    public class TreeRegressorCollection
+    {
+        [BestFriend]
+        internal readonly TreeEnsemble TreeEnsemble; // It's a best friend for being accessed from LightGBM.
```

Comment:
>It's a best friend for being accessed from LightGBM. [](start = 56, length = 52)

convert to xml comment?

Reply:
Sure.

---
In reply to: [251969703](https://github.com/dotnet/machinelearning/pull/2243#discussion_r251969703) [](ancestors = 251969703)